### DISCUSSION TITLE
Is there a way to remap the , to a . in the floating point display mode?

### DISCUSSION THREAD

**Discussion by marklacas** [tabemann/zeptoforth#20] [Q&A] [answered]
Is there a way to remap the , to a . in the floating point display mode?

**Comment by tabemann** [comment, marked as answer]
This can be achieved by changing the following code in `src/common/forth/basic.fs` at line 2183:

    \ Add a decimal point
    : add-decimal ( c-addr bytes -- c-addr bytes )
      2dup + [char] , swap c! 1+
    ;

to:

    \ Add a decimal point
    : add-decimal ( c-addr bytes -- c-addr bytes )
      2dup + [char] . swap c! 1+
    ;

and then following the build instructions in `README.md`.

Note that changing the decimal point to `.` when *parsing* code is less feasible, because it is already used for parsing non-fixed point double-cell numbers, by Forth tradition.

**Reply by marklacas** [reply]
Thank you!
